CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ cmake_minimum_required(VERSION 3.0)
 
 project(CpuFeatures VERSION 0.1.0)
 
+set(CMAKE_C_STANDARD 99)
```

Comment:
I'm not super proficient at CMake.
Is this going to be contaminating? I mean if a project uses `cpu_features` will it become C99 as well?

Reply:
It sets default C_STANDARD for all targets in the file. So, it wouldn't be visible in parent CMakeFile.